src/components/Sandbox.js

```diff
@@ -11,41 +14,103 @@ const styles = (theme) => ({
     flexGrow: 1,
     padding: 8,
   },
+  chordSourcePaper: {
+    margin: theme.spacing.unit,
+    padding: theme.spacing.unit * 2,
+  },
+  chordCategoryPaper: {
+    margin: theme.spacing.unit,
+    textAlign: 'center',
+  },
+  chordCategoryTypography: {
+    color: theme.palette.text.primary,
+    textAlign: 'center',
+  },
+  chordVariantPaper: {
+    display: 'inline-block',
+    margin: theme.spacing.unit,
+    textAlign: 'center',
+  },
+  chordVariantTypography: {
+    color: theme.palette.text.secondary,
+  },
+  chordDiagram: {
+    filter: theme.palette.type === 'dark' ? 'invert(100%)' : '',
+  }
 });
 
-const Sandbox = ({ classes, match }) => (
+const Sandbox = ({ classes, location, match }) => (
   <div className={classes.root}>
     <Typography variant='h2'>Sandbox</Typography>
 
     <ul>
       <li><Link to={`${match.url}/california-dreamin`}>California Dreamin' - The Mamas and the Papas</Link></li>
       <li><Link to={`${match.url}/the-giving-tree`}>The Giving Tree - Plain White T's</Link></li>
-      <li><Link to={`${match.url}/chord-library`}>Chord Library</Link></li>
+      <li><Link to={`${match.url}/riptide-coolkids-mashup`}>Riptide / Coolkids Mashup</Link></li>
+      <li><Link to={`${match.url}/guitar-chordbook`}>Guitar Chordbook</Link></li>
     </ul>
 
     <Route path={`${match.path}/california-dreamin`}
-      component={() => <MusicMarkdown source={californiaDreaminSource} />} />
+      component={() => <MarkdownViewer location={location} source={californiaDreaminSource} />} />
```

Comment:
Maybe at some point in the future, we should just move these to another sandbox directory. Nit.

Reply:
Agree.